peer/peerstorage_test.go

```diff
@@ -0,0 +1,69 @@
+package peer
+
+import (
+	"os"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/lightningnetwork/lnd/kvdb"
+	"github.com/stretchr/testify/require"
+)
+
+// TestKvdbPeerStorage tests the KvdbPeerStorage functionality.
+func TestKvdbPeerStorage(t *testing.T) {
+	rt := require.New(t)
+
+	file, err := os.CreateTemp("", "*.db")
+	rt.NoError(err, "unable to create temp file")
+	t.Cleanup(func() {
+		rt.NoError(file.Close())
+		rt.NoError(os.Remove(file.Name()))
+	})
+
+	dbPath := file.Name()
+	db, err := kvdb.Open(
+		kvdb.BoltBackendName, dbPath, true, kvdb.DefaultDBTimeout,
+	)
+	t.Cleanup(func() {
+		rt.NoError(db.Close())
+	})
+
+	rt.NoError(err, "unexpected err creating kvdb DB")
```

Comment:
personally I dont think you need to add strings to these assertions. If the assertion fails, it will print out a line number that will point us to what went wrong. Easier to read if it is just `rt.NoError(err)` imo

Reply:
Thanks. I will like to understand what makes it easier to read seeing that the error strings serves as a form of documentation/comment.

Reply:
it seems redundant. The test will point to where the error occurred. So it will basically say _twice_ that an error occurred here. 

Not blocking of course but I think all the text distracts from the test itself a bit.

Reply:
I think that unless you plan on putting other context elements (displaying the values of variables currently in scope), elle is correct that it is redundant.